docs/docsite/rst/index.rst

```diff
@@ -61,6 +61,12 @@ Ansible releases a new major release of Ansible approximately every two months.
    scenario_guides/guide_vagrant
    scenario_guides/guide_vmware
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Ansible for VMWare
+
+   vmware/index
+
```

Comment:
I think the VMWare guide should go under Scenario Guides, but the way this is written I think it will be added as a new section of the left nav bar. I'd like to keep the left nav as small as possible. It gets generated on every page, so each addition slows the docs build. I'll build the docs from the branch and confirm, but let me know what you were thinking for the TOC/nav.

Reply:
Oh, jeez, wrong PR. This one is already merged.